Orange/widgets/classify/owadaboost.py

```diff
@@ -28,12 +28,12 @@ class OWAdaBoostClassification(OWBaseLearner):
     def add_main_layout(self):
         box = gui.widgetBox(self.controlArea, "Parameters")
         self.base_estimator = TreeLearner()
-        self.base_label = gui.label(box, self, "Base estimator: " + self.base_estimator.name)
+        self.base_label = gui.label(box, self, "Base Estimator: " + self.base_estimator.name)
 
-        gui.spin(box, self, "n_estimators", 1, 100, label="Number of estimators",
+        gui.spin(box, self, "n_estimators", 1, 100, label="Number of Estimators",
```

Comment:
I don't think I like Title Case everywhere. Perhaps for section titles; [definitely not for option labels](https://www.google.si/search?tbm=isch&q=libreoffice+calc+preferences&imgrc=Vf_5Y57gkjehSM%3A&cad=h)!